docs/opentelemetry.asciidoc

```diff
@@ -0,0 +1,182 @@
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/java[elastic.co]
+endif::[]
+
+[[opentelementry-bridge]]
+=== OpenTelemetry bridge
+
+NOTE: Added as experimental in 1.21.0.
+To enable it, set <<config-enable-experimental-instrumentations, `enable_experimental_instrumentations`>> to `true`.
+
+The Elastic APM OpenTelemetry bridge allows creating Elastic APM `Transactions` and `Spans`,
+using the OpenTelemetry API.
+In other words,
+it translates the calls to the OpenTelemetry API to Elastic APM and thus allows for reusing existing instrumentation.
+
+The first span of a service will be converted to an Elastic APM
+{apm-overview-ref-v}/transactions.html[`Transaction`],
+subsequent spans are mapped to Elastic APM
+{apm-overview-ref-v}/transaction-spans.html[`Span`].
+
+[float]
+[[otel-operation-modes]]
+==== Operation Modes
+
+This bridge allows for different operation modes in combination with the Elastic APM `javaagent`
+
+Noop::
++
+--
+If the `javaagent` is not specified, the bridge is in noop mode and does not actually record and report spans.
+--
+
+Mix and Match::
++
+--
+If you want to leverage the auto instrumentation of Elastic APM,
+but also want to create custom spans or use the OpenTelemetry API to add custom tags to the spans created by Elastic APM,
+you can just do that.
+The OpenTelemetry bridge and the standard Elastic APM API interact seamlessly.
+--
+
+Manual instrumentation::
++
+--
+If you don't want Elastic APM to auto-instrument known frameworks,
+but instead only rely on manual instrumentation,
+disable the auto instrumentation setting the configuration option <<config-instrument,`instrument`>> to `false`.
```

Comment:
[minor] to make that easier to read, maybe keep the same structure for all of the 3 cases

*Noop*
- OpenTelemetry API, Elastic APM API or both: included in application dependencies
- Elastic APM Agent: not attached to the JVM

Usage: deploy without the agent (no hard dependency on it)

*Mix and Match*
- OpenTelemetry API, Elastic APM API or both: included in application dependencies
- Elastic APM Agent: attached to the JVM, auto-instrumentation enabled

Usage: Using both OpenTelemetry API, Elastic APM API and automatic instrumentations

*Manual instrumentation*
- OpenTelemetry API, Elastic APM API or both: included in application dependencies
- Elastic APM Agent: attached to the JVM, auto-instrumentation disabled

Usage: Manual instrumentation using either OpenTelemetry or Elastic APM API

Reply:
Are you saying this part should be removed from the OTel and OTracing docs and bubble up to apis.asciidoc?

Reply:
Yes, that would be a great idea, so we could only keep things related to OTel here.

Reply:
Done